src/Microsoft.AspNetCore.Mvc.Core/BindPropertiesAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// An attribute that enables binding for all properties the decorated controller or Razor Page model defines.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
+    public class BindPropertiesAttribute : Attribute
+    {
+        /// <summary>
+        /// When <c>true</c>, allows properties to be bound on GET requests.
```

Comment:
Please also add comment describing the behavior for the opposite (false) case, which is the default.